src/shared/commons/form-group-helper.ts

```diff
@@ -0,0 +1,63 @@
+import { AbstractControl, FormArray, FormGroup } from '@angular/forms';
+import { CollectionsHelpers } from './collections-helpers';
+
+export class FormGroupHelper {
```

Comment:
mark this as `@Injectable()` so that you can inject it where needed rather than having every time to create new instances. See for example `FieldsUtils`

Reply:
OK

Reply:
if I create a Bean for that I will have change the @Module test coverage  for this project as well as CaseManagement hence another pr and new release for CaseManagement .